deploy/ansible/main.yml

```diff
@@ -1,13 +1,24 @@
 ---
-  - hosts: all
-    gather_facts: no
+- hosts: ubuntu
+  gather_facts: yes
     pre_tasks:
-
       - name: install python 2  # python installation (ansible hosts gather_facts requirement)
         raw: test -e /usr/bin/python || (apt -y update && apt install -y python)
         changed_when: False
 
-    roles:
-      - { role: iroha }
-      - { role: docker }
-      - { role: notary }
+- hosts: docker
+  gather_facts: yes
+  pre_tasks:
+    - name: install python 2
+      raw: tce-load -wi python
+      become: false
+
+    - name: Install pip
```

Comment:
Pip can be installed inside main `tasks` file. This package is not essential for Ansible operations. And, please, install it via `apt` Ansible module.